packages/ts-migrate-plugins/src/plugins/react-props.ts

```diff
@@ -134,34 +135,54 @@ function updatePropTypes(
 
   if (isReactSfcNode(node)) {
     const propsParam = getPropsParam(node);
+    const forwardRefComponent = getReactForwardRefFuncExpression(node);
+
     if (propsParam && !propsParam.type) {
       const propTypesNode = findSfcPropTypesNode(node, sourceFile);
       const objectLiteral = propTypesNode && findPropTypesObjectLiteral(propTypesNode, sourceFile);
       if (objectLiteral) {
         updates.push(
           ...updateObjectLiteral(node, objectLiteral, propsTypeName, sourceFile, options, false),
         );
-
-        updates.push({
-          kind: 'replace',
-          index: propsParam.pos,
-          length: propsParam.end - propsParam.pos,
-          text: printer.printNode(
-            ts.EmitHint.Unspecified,
-            ts.updateParameter(
-              propsParam,
-              propsParam.decorators,
-              propsParam.modifiers,
-              propsParam.dotDotDotToken,
-              propsParam.name,
-              propsParam.questionToken,
-              ts.createTypeReferenceNode(propsTypeName, undefined),
-              propsParam.initializer,
+        if (forwardRefComponent) {
```

Comment:
instead of casting `forwardRefComponent as any` and a result below, you can just update this check to:
```
if (forwardRefComponent !== false && ts.isExpressionWithTypeArguments(forwardRefComponent)) {   
  const forwardRef = forwardRefComponent as ts.ExpressionWithTypeArguments;
```
and use `forwardRef` instead of `forwardRefComponent`

Reply:
this seems to break the current implementation, as calling `ts.isExpressionWithTypeArguments(forwardRefComponent)` results in `false`. I suppose that is happening because the initial `CallExpression` has no typings whatsoever and hence the above function check just refuses to accept it